holoviews/plotting/bokeh/__init__.py

```diff
@@ -54,7 +54,7 @@
 
                 # Rasters
                 Image: RasterPlot,
-                GridImage: ImagePlot,
+                GridImage: RasterPlot,
```

Comment:
There is another issue removing GridImage, users will have used ``GridImage`` as an option. Is there any way to add Element option aliases, i.e. it would apply options set on ``GridImage`` to ``Image``, I vaguely recall something like that. That may be a good place to put a deprecation warning too.